doc/nrf/device_guides/working_with_nrf/nrf53/nrf5340.rst

```diff
@@ -584,6 +584,11 @@ Bluetooth buffers configuration introduced by the :kconfig:option:`CONFIG_NCS_SA
 Simultaneous multi-image DFU
 ****************************
 
+.. note::
```

Comment:
We don't usually have "Notes" immediately after a heading title (I could be wrong about this?)
I would suggest to move this to  after line 594.

Or, if this info is very important to have it here in the beginning I would take it out of the "note".

Reply:
This suggestion is OK, but now we have two notes one after another, which sometimes is a bit of an overkill.

Reply:
> This suggestion is OK, but now we have two notes one after another, which sometimes is a bit of an overkill.

Yeah I agree. I wonder, whether instead of the note there should be rather some introduction chapter, on the page, that would remind on how the Kconfig options work with child-images and how to address child images, instead of doing remark here.

It kinda reminds me what Larry Trusk has written about footnotes in "Punctuation" (https://www.sussex.ac.uk/informatics/punctuation/essaysandletters/footnotes); in short:
```
 There are two main rules in the use of footnotes. First:

    Do not use a footnote if you can possibly avoid it. 
```

Reply:
That is a good point, as too many notes lead to users ignoring them. I don't oppose the idea of the introduction, but in this particular case I also think that the ``.. note::`` is not needed and this information can leave as normal text in the new location after the list of Kconfigs.

Reply:
OK, I will remove the `.. note::` and make it part of text.